src/jdmtool/data_card/skybound.py

```diff
@@ -18,6 +17,14 @@ class SkyboundDevice(ProgrammingDevice):
 
     is_orange_card = False
 
+    # Standard values: WRITE_ENDPOINT = 0x02, READ_ENDPOINT = 0x81
+    def __init__(self, handle: USBDeviceHandle, read_endpoint: int = 0x81, write_endpoint: int = 0x02) -> None:
+        # Initialize base device
+        super().__init__(handle)
+        # Override endpoints for this Skybound device
+        self.READ_ENDPOINT = read_endpoint
+        self.WRITE_ENDPOINT = write_endpoint
+
```

Comment:
`read_endpoint` and `write_endpoint` should probably become part of the super-class - same as `handle`.

I can also refactor this myself first, if that makes it easier.